packages/ui-react/src/components/DeleteAccountModal/DeleteAccountModal.tsx

```diff
@@ -12,18 +12,21 @@ import PasswordField from '../PasswordField/PasswordField';
 import Button from '../Button/Button';
 import Alert from '../Alert/Alert';
 import { modalURLFromLocation } from '../../utils/location';
+import { useAriaAnnouncer } from '../../containers/AnnouncementProvider/AnnoucementProvider';
 
 import styles from './DeleteAccountModal.module.scss';
 
 const DeleteAccountModal = () => {
   const accountController = getModule(AccountController);
 
   const { t } = useTranslation('user');
+  const announcer = useAriaAnnouncer();
```

Comment:
If you want to be correct, then we should always name it like you did in Account.tsx: :-)
`const announce = useAriaAnnouncer();`